server/package.json

```diff
@@ -37,5 +37,9 @@
     "passport": "^0.5.2",
     "passport-local": "^1.0.0",
     "pg": "^8.7.1"
+  },
+  "devDependencies": {
+    "dotenv": "^8.2.0",
+    "nodemon": "^2.0.7"
```

Comment:
just make sure to get the updates I posted in slack in the future in your codebase!